### PR TITLE
Added project and freeze plan for 4.27 (without dates)

### PR DESCRIPTION
**Pull request by iloveeclipse**

### CODE REVIEW

**Comment by iloveeclipse** [comment]
@akurtakov : no exact dates added because I don't have an idea how they are computed.

File: development/plans/eclipse_project_plan_4_27.xml [hunk 1]
```diff
@@ -0,0 +1,454 @@
...
+
+			<td rowspan="2">9.0
+			</td>
+			<td rowspan="1">
+				x86 64-bit<br/>
+				aarch64<br/>
+			</td>
+			<td rowspan="1">
+				OpenJDK 11.0.15 (LTS)<br/>
+				OpenJDK 17.0.3 (LTS)<br/>
```
Comment (akurtakov) [inline development/plans/eclipse_project_plan_4_27.xml:226]: 17.0.5

File: development/plans/eclipse_project_plan_4_27.xml [hunk 2]
```diff
@@ -0,0 +1,454 @@
...
+				Oracle Java 11.0.15 (LTS)<br/>
+				Oracle Java 17.0.3 (LTS)<br/>
```
Comment (akurtakov) [inline development/plans/eclipse_project_plan_4_27.xml:228]: 17.0.5

File: development/plans/eclipse_project_plan_4_27.xml [hunk 3]
```diff
@@ -0,0 +1,454 @@
...
+		<!-- ************ SLES ************** -->
+		<tr class="c0">
+			<td rowspan="2">SUSE Linux Enterprise Server</td>
+			<td rowspan="2">
+				15 SP4
+			</td>
+			<td rowspan="1">x86 64-bit</td>
+			<td rowspan="1">
+				OpenJDK 11.0.15 (LTS)<br/>
+				OpenJDK 17.0.3 (LTS)<br/>
```
Comment (akurtakov) [inline development/plans/eclipse_project_plan_4_27.xml:247]: 17.0.5

File: development/plans/eclipse_project_plan_4_27.xml [hunk 4]
```diff
@@ -0,0 +1,454 @@
...
+		<tr class="c1">
+			<td rowspan="1">Ubuntu Long Term Support</td>
+		    <td rowspan="1">22.04</td>
+			<td rowspan="1">
+				x86 64-bit<br/>
+				aarch64<br/>
+			</td>
+			<td rowspan="1">
+				OpenJDK 11.0.15 (LTS)<br/>
+				OpenJDK 17.0.3 (LTS)<br/>
```
Comment (akurtakov) [inline development/plans/eclipse_project_plan_4_27.xml:268]: 17.0.5

**Comment by iloveeclipse** [comment]
Alex, just push your changes if there are mkre

**Comment by akurtakov** [comment]
I'll adjust versions accordingly but I think we should drop Java 11 as target environment as we are no longer testing on it.

**Comment by iloveeclipse** [comment]
I've created https://github.com/eclipse-platform/eclipse.platform.releng/issues/189 for the rest.
If you now how to compute dates, please add that to the plan.
Thanks.

**Pull request merged**